src/Compiler/CompilerUtils.cpp

```diff
@@ -532,6 +532,14 @@ static void genLLVMBitcode(const mlir::OwningOpRef<ModuleOp> &module,
     }
   }
 
+  // Emit the onnx-mlir version as llvm.ident metadata.
+  llvm::NamedMDNode *identMetadata =
+      llvmModule->getOrInsertNamedMetadata("llvm.ident");
+  std::string version = "onnx-mlir version 1.0.0";
```

Comment:
According to our release notes, the latest version is 0.2.0 not 1.0.0. But given that this version is for the current code, I recommend to include git commit id which is more useful. For example, in clang, I see this
```
!llvm.ident = !{!5}
!5 = !{!"clang version 14.0.0 (git@github.ibm.com:compiler/llvm-project.git 033b8a4be1acb95d46ba510c7329977ceb80a680)"}
```

Can we have a global place for the version number? I believe we should make the version info consistent across the whole code. At least, a global variable in CompilerUtils.cpp would be helpful.

Reply:
I spoke to @whitneywhtsang to do this in a separate PR.

Reply:
https://github.com/onnx/onnx-mlir/pull/1215